project/BuildHelper.scala

```diff
@@ -151,7 +152,7 @@ object BuildHelper {
   def stdSettings(prjName: String) =
     Seq(
       name := s"$prjName",
-      crossScalaVersions := Seq(Scala212, Scala213),
+      crossScalaVersions := Seq(Scala212, Scala213, Scala3),
```

Comment:
I think you may want to take a look at `extraOptions` settings. Currently, scala3 would be "ignored". `zio-prelude` and `zio` have a complain setup that you could reuse.